Makefile

```
@@ -542,3 +542,18 @@ fix-imports: sort-imports ## Sort imports
+
+# Shared dev environment
+# Uses a local sibling checkout if available (e.g. ../tools),
+# otherwise shallow-clones the tools repo into .tools/ on first use.
+TOOLS_DIR ?= $(shell cd .. && pwd)/tools
+DEV_MK := $(TOOLS_DIR)/dev/dev.mk
+ifeq ($(wildcard $(DEV_MK)),)
+  TOOLS_DIR := $(shell pwd)/.tools
+  DEV_MK := $(TOOLS_DIR)/dev/dev.mk
+  ifeq ($(wildcard $(DEV_MK)),)
+    $(info Downloading medik8s/tools into .tools/...)
+    $(shell git clone --depth 1 https://github.com/medik8s/tools.git $(TOOLS_DIR) >/dev/null 2>&1)
```

checkmake (0.3.2)

[warning] Target "$(shell git clone --depth 1 https" should be declared PHONY.

(phonydeclared)